### ISSUE TITLE
Good simple code, what are your accuracies?

### ISSUE THREAD

**Issue by christiaan20** [AaronWard/Convolutional-neural-network-for-image-recognition#1]
This is good simple code, thank you for uploading it!

I only changed the the steps per epoch to 8000/32 for the training and 2000/32 for the test set and saved the model that is generated.

I'd love to know what kind of accuracy you got, I got the following:
for training set loss is 0.211 and the accuracy is 0.915
for test set loss is 0.531 and the accuracy is 0.797
This is better then most machine learning code I have tested up until now.

The only test it didn't pass is when I inputted a picture of my own cat and even when inputting pictures of cats from the internet where the algorithm will nearly always say 100 % dog. So I'll have to start teach my cat some dog tricks.
